components/org.wso2.micro.integrator.initializer/src/main/java/org/wso2/micro/integrator/initializer/ServiceBusInitializer.java

```diff
@@ -165,6 +175,20 @@ protected void activate(ComponentContext ctxt) {
 
                 synapseEnvironment.registerSynapseHandler(new SynapseExternalPropertyConfigurator());
                 synapseEnvironment.registerSynapseHandler(new ProxyLogHandler());
+
+                // Register internal transaction synapse handler
+                boolean transactionPropertyEnabled = TransactionHandlerComponent.isTransactionPropertyEnabled();
+                if (transactionPropertyEnabled) {
+                    TransactionHandler transactionHandler = new TransactionHandler();
```

Comment:
Will the word requestCountHandler/TransactionCountHandler be clear?

Reply:
+1. `TransactionHandler` will mean a component actually handling the transactions.

Reply:
Changed.